README.md

```diff
@@ -8,15 +8,21 @@
 [![Playground](https://shields.io/badge/JupyterLab-Enjoy%20Your%20FL%20Journey!-F37626?logo=jupyter)](https://try.federatedscope.io/)
 [![Contributing](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://federatedscope.io/docs/contributor/)
 
-FederatedScope is a comprehensive federated learning platform that provides convenient usage and flexible customization for various federated learning tasks in both academia and industry.  Based on an event-driven architecture, FederatedScope integrates rich collections of functionalities to satisfy the burgeoning demands from federated learning, and aims to build up an easy-to-use platform for promoting learning safely and effectively.
+![Github Repo Size](https://img.shields.io/github/repo-size/alibaba/FederatedScope?style=for-the-badge&color=aqua)
 
-A detailed tutorial is provided on our website: [federatedscope.io](https://federatedscope.io/)
+FederatedScope is a **comprehensive federated learning platform** that provides convenient usage and flexible customization for various federated learning tasks in both academia and industry.  Based on an event-driven architecture, it integrates rich collections of functionalities to satisfy the burgeoning demands from federated learning, and **aims to build up an easy-to-use platform** for promoting learning safely and effectively.
+
+A [detailed tutorial](https://federatedscope.io/) is provided on our website.
 
 You can try FederatedScope via [FederatedScope Playground](https://try.federatedscope.io/) or [Google Colab](https://colab.research.google.com/github/alibaba/FederatedScope).
 
+
+|||||||
+|--|--|--|--|--|--|
```

Comment:
It seems that there exists a blank line in the table and please modify it

Reply:
It is the basic table structure for making a table in README.